src/test/integration/tokens.e2e-spec.ts

```diff
@@ -111,4 +123,73 @@ describe('Token Service', () => {
       expect(await tokenService.getToken(tokenIdentifier + 'a')).toBeUndefined();
     });
   });
-});
+
+  describe('Get Token Roles', () => {
+    it(`should return token roles`, async () => {
+      const tokenRoles = await tokenService.getToken(tokenIdentifier);
+
+      if (tokenRoles) {
+        const value = await tokenService.getTokenRoles(tokenRoles.identifier);
+        expect(value).toBeInstanceOf(Array);
+      }
+    });
+  });
+
+  describe('Get Token Accounts', () => {
+    it(`should return token with size of 10`, async () => {
+      const tokenRoles = await tokenService.getToken(tokenIdentifier);
+
+      if (tokenRoles) {
+        const tokensList = await tokenService.getTokenAccounts({from: 0, size: 10}, tokenRoles.name);
+        expect(tokensList).toBeInstanceOf(Array);
+      }
+    });
+  });
+
+  describe('Get Token For Address', () => {
```

Comment:
Also, for this context an account address should be set

Reply:
Change request resolved